lms/djangoapps/mobile_api/course_info/tests.py

```diff
@@ -84,60 +84,95 @@ def test_updates(self, new_format):
             self.assertIn("Update" + str(num), update_data['content'])
 
 
+@ddt.ddt
 class TestHandouts(MobileAPITestCase, MobileAuthTestMixin, MobileCourseAccessTestMixin, MilestonesTestCaseMixin):
     """
     Tests for /api/mobile/v0.5/course_info/{course_id}/handouts
     """
     REVERSE_INFO = {'name': 'course-handouts-list', 'params': ['course_id']}
 
-    def setUp(self):
-        super(TestHandouts, self).setUp()
-
-        # Deleting handouts fails with split modulestore because the handout has no parent.
-        # This needs further investigation to determine if it is a bug in the split modulestore.
-        # pylint: disable=protected-access
-        self.store = modulestore()._get_modulestore_by_type(ModuleStoreEnum.Type.mongo)
-
-        # use toy course with handouts, and make it mobile_available
-        course_items = import_course_from_xml(self.store, self.user.id, settings.COMMON_TEST_DATA_ROOT, ['toy'])
+    @ddt.data(ModuleStoreEnum.Type.mongo, ModuleStoreEnum.Type.split)
+    def test_handouts(self, default_ms):
+        with self.store.default_store(default_ms):
+            self.add_mobile_available_toy_course()
+            response = self.api_response(expected_response_code=200)
+            self.assertIn("Sample", response.data['handouts_html'])
```

Comment:
See earlier comment.  This should be implemented inside `verify_success`, according to this module's test configuration using the `MobileAPITestCase` base class.